Index: s6-2.0.0.0/Makefile
===================================================================
--- s6-2.0.0.0.orig/Makefile	2014-12-31 02:56:33.895224764 +0100
+++ s6-2.0.0.0/Makefile	2014-12-31 02:57:00.623224981 +0100
@@ -114,7 +114,7 @@
 	exec $(RANLIB) $@
 
 lib%.so:
-	exec $(REALCC) -o $@ $(CFLAGS_ALL) $(CFLAGS_SHARED) $(LDFLAGS_ALL) $(LDFLAGS_SHARED) -Wl,-soname,$@.$(version_l) $^
+	exec $(REALCC) -o $@ $(CFLAGS_ALL) $(CFLAGS_SHARED) $(LDFLAGS_ALL) $(LDFLAGS_SHARED) -Wl,-soname,$@.$(version_l) $^ -lskarnet
 
 .PHONY: it all clean distclean tgz strip install install-dynlib install-bin install-sbin install-lib install-include
 
